include/radproto/vendor_attribute.h

```diff
@@ -5,18 +5,22 @@
 #include <vector>
 #include <cstdint> //uint8_t, uint32_t
 
-class VendorSpecific
+namespace RadProto
 {
-    public:
-        VendorSpecific(const uint8_t* data);
-        VendorSpecific(uint32_t vendorId, uint8_t vendorType, const std::vector<uint8_t>& vendorValue);
-        std::string toString() const;
-        uint8_t vendorType() const { return m_vendorType; }
-        uint32_t vendorId() const { return m_vendorId; }
-        std::vector<uint8_t> toVector() const;
-    private:
-        uint32_t m_vendorId;
-        uint8_t m_vendorType;
-        std::vector<uint8_t> m_value;
-};
+    class VendorSpecific
```

Comment:
There is a problem with this class. Consider your sample server reply:
```
Sent Access-Request Id 117 from 0.0.0.0:60875 to 127.0.0.1:1812 length 74
        User-Name = "test"
        User-Password = "123456"
        NAS-IP-Address = 127.0.0.1
        NAS-Port = 1
        Message-Authenticator = 0x00
        Cleartext-Password = "123456"
Received Access-Accept Id 117 from 127.0.0.1:1812 to 127.0.0.1:60875 length 95
        User-Name = "test"
        NAS-Port = 20
        NAS-IP-Address = 127.104.22.17
        User-Password = "password123"
        Callback-Number = "123abc"
        CHAP-Password = 0x0131323334353637383961626364656667
        Dlink-User-Level = 808452096
```

You can observe that 'Dlink-User-Level' is rendered as an integer value (it is indeed integer according to the dictionary). You set it as the following vector of bytes:
```c++
std::vector<uint8_t> vendorValue {'0', '0', '0', '3'};
```
It looks strange because you use digits as characters rather than values (i.e., `{0, 0, 0, 3}`), but its okay. If we take these bytes and turn them into a big endian value, we get:
```
 - '0' has hex code 0x30 = 48;
 - '3' has hex code 0x33 = 51.
 
 48 * 256 * 256 * 256 +
 48 * 256 * 256 +
 48 * 256 +
 51 = 808464435 = 0x30303033
```
However, your sample server responds with 808452096 = 0x30300000

It seems that your `toVector` code takes only first two bytes from the value and skips the rest.